openmdao/drivers/genetic_algorithm_driver.py

```diff
@@ -243,6 +251,37 @@ def objective_callback(self, x, icase):
         """
         Evaluate problem objective at the requested point.
 
+        Takes into account constraints with a penalty function.
+
+        All constraints are converted to the form of :math:`g(x)_i \leq 0` for inequality constraints and
+        :math:`h(x)_i = 0` for equality constraints.
+        The constraint vector for inequality constraints is the following:
+
+        .. math::
+
+           g = [g_1, g_2  \dots g_N], g_i \in R^{N_{g_i}}
+
+           h = [h_1, h_2  \dots h_N], h_i \in R^{N_{h_i}}
+
+        The number of all constraints:
+
+        .. math::
+
+            N_g = \sum_{i=1}^N N_{g_i},  N_h = \sum_{i=1}^N N_{h_i}
+
+        The fitness function is constructed with the penalty parameter :math:`p` and the exponent :math:`\kappa`:
+
```

Comment:
The Integration tests are mostly failing due to PEP standard issues. Here a few of these lines are too long.  If you take them down to under 100 it'll be fine.